src/ngraph/runtime/intelgpu/intelgpu_op_custom_kernels.cpp

```diff
@@ -823,55 +823,36 @@ void runtime::intelgpu::do_logic_kernel(cldnn::topology& topology,
     const cldnn::layout layout = IntelGPULayout::create_cldnn_layout(output_type, output_shape);
     const string entry_point_name = "logic_" + output_name;
     codegen::CodeWriter writer;
+    vector<size_t> gws;
 
-    writer << "__kernel void " << entry_point_name << "(const __global " << inputA_type << " inputA"
-           << array_dims(inputA_shape) << ", const __global " << inputB_type << " inputB"
-           << array_dims(inputB_shape) << ", __global char output" << array_dims(output_shape)
-           << ")\n";
+    runtime::intelgpu::gen_func_def(writer,
+                                    entry_point_name,
+                                    {2, input0_type},
```

Comment:
Are you sure that types will be the same in all cases?
I mean, may be better to use `{input0_type, input1_type}`

Reply:
According to docs, e.g. http://ngraph.nervanasys.com/docs/latest/ops/greater_eq.html or http://ngraph.nervanasys.com/docs/latest/ops/and.html or http://ngraph.nervanasys.com/docs/latest/ops/equal.html